src/System.Numerics.Experimental/Half.cs

```diff
@@ -341,17 +353,317 @@ public override int GetHashCode()
 
         public override string ToString()
         {
-            return ToString(format: null, formatProvider: null);
+            return $"0x{m_value:X4}";
+            // return ToString(format: null, formatProvider: null);
+            // TODO: Implement this
         }
 
         public string ToString(string format = null, IFormatProvider formatProvider = null)
         {
-            throw new NotImplementedException();
+            return $"0x{m_value:X4}";
+            // throw new NotImplementedException();
+            // TODO: Implement this
         }
 
         public bool TryFormat(Span<char> destination, out int charsWritten, ReadOnlySpan<char> format, IFormatProvider formatProvider)
         {
             throw new NotImplementedException();
         }
+
+        // -----------------------Start of to-half conversions-------------------------
+
+        public static implicit operator Half(int value)
+        {
+            bool sign = value < 0;
+            Half h = (uint)(sign ? -value : value); // Math.Abs but doesn't throw exception, because we cast it to uint anyway
+            return sign ? new Half((ushort)(h.m_value | SignMask)) : h;
+        }
+
+        public static implicit operator Half(uint value)
+        {
+            int shiftDist = BitOperations.LeadingZeroCount(value) - 21;
+            if (shiftDist >= 0)
```

Comment:
nit: Using braces makes the scope much clearer

Reply:
I'm that 'no brace if possible' guy... but sure. 😁

Do you want me to do it for every `if`s in general, or only for these multi-line statements?

Reply:
The only hard rule (see https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md and quoted below), is:
> A single line statement block can go without braces but the block must be properly indented on its own line and must not be nested in other statement blocks that use braces (See issue 381 for examples).

My personal preference is to always use braces (I think it helps readability and can make diffs smaller when adding code/asserts/etc), but I won't block the PR on it 😄

Reply:
Braces everywhere it is.